build.cake

```diff
@@ -766,12 +766,31 @@ Task("PackageZip")
         Zip(currentImageDir, File(ZIP_PACKAGE_SL), zipFiles);
     });
 
-Task("PackageMsi")
+
+Task("PackageRunnersMsi")
```

Comment:
Sure - `Runners` was the name for the existing engine/console msi. If we do remove the framework msi (see above) - we could just revert to PackageMsi?

Runners-Msi is also used in the file structure for now to distinguish from framework-msi for now - these can perhaps be renamed again once the repos are split?